hawkular-wildfly-agent/pom.xml

```diff
@@ -113,6 +113,7 @@
     <dependency>
       <groupId>org.jboss.logging</groupId>
       <artifactId>jboss-logging</artifactId>
+      <version>3.1.4.GA-redhat-2</version>
```

Comment:
This one is worrying because we are setting a hardcoded version but yet are saying it is "provided". So we are not guaranteed the agent is going to run with this version of logging (it will be provided with whatever the server has).

Why do we need to hardcode this specific redhat version? And what happens if we run the agent in a server that doesn't have this version?